lib/loga/sidekiq/job_logger.rb

```diff
@@ -0,0 +1,45 @@
+module Loga
+  module Sidekiq
+    class JobLogger
+      include Loga::Utilities
+
+      EVENT_TYPE = 'sidekiq_job_logger'.freeze
+
+      attr_reader :start
+
+      def initialize
+        @start = Time.now
+      end
+
+      def call(item, _queue)
+        event = ->(note) { Event.new(data: item, message: note, type: EVENT_TYPE) }
+
+        ::Sidekiq::Logging.with_context("source=#{item['class']}") do
+          begin
+            logger.info(event.call('start'))
+            yield
+            logger.info(event.call(declare_success))
+          rescue Exception # rubocop:disable Lint/RescueException
+            logger.info(event.call(declare_failure))
+
+            raise
+          end
+        end
+      end
+
+      private
+
+      def declare_success
+        "count#job.success=1 measure#job.duration=#{duration_in_ms(start)}ms"
```

Comment:
Just log a loga event like in the rack logger https://github.com/FundingCircle/loga/blob/master/lib/loga/rack/logger.rb#L51

Make sure to add:
-  `data['duration'] = ...`
- `data['worker'] = item['class']`